__tests__/app.test.js

```diff
@@ -67,13 +67,32 @@ describe("/api/articles", () => {
         });
       });
   });
-  test("GET:200 articles are sorted by date in descending order", () => {
+
+  test("GET:200 articles are sorted by date in descending order (by default)", () => {
     return request(app)
       .get("/api/articles")
       .expect(200)
-      .then(({ body: { articles } }) =>
-        expect(articles).toBeSortedBy("created_at", { descending: true }),
-      );
+      .then(({ body: { articles } }) => {
+        expect(articles.length).toBe(13);
+        expect(articles).toBeSortedBy("created_at", { descending: true });
+      });
+  });
+
+  test("GET:200 articles are sorted by any valid column", () => {
+    return request(app)
+      .get("/api/articles?sort_by=title&&order=asc")
+      .expect(200)
+      .then(({ body: { articles } }) => {
+        expect(articles.length).toBe(13);
+        expect(articles).toBeSortedBy("title", { descending: false });
+      });
+  });
```

Comment:
This is great and it proves with a good level of confidence that both queries are working, it could be good to separate them out into two separate tests one for sort and one for order to get even more confidence 🤔